### PR TITLE
[DEV-1204] og tags

### PR DESCRIPTION
**Pull request by tommaso1**
#### List of Changes

Add meta tags to home page and replace svg with images for a better include card on social networks

#### How Has This Been Tested?

Using this [tool](https://www.opengraph.xyz/) and by sharing a page on social networks personal accounts

#### Screenshots (if appropriate):
image
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/app/layout.tsx [hunk 1]
```diff
@@ -73,6 +73,29 @@ export default async function RootLayout({
...
+        <meta
+          name='twitter:description'
+          content='Il portale per gli sviluppatori di PagoPA'
+        />
+        <meta
+          name='twitter:image'
+          content='https://developer.pagopa.it/images/dev-portal-home.jpg'
+        />
+        <meta name='twitter:image:alt' content='PagoPA DevPortal' />
+        <meta name='twitter:creator' content='@pagopa' />
```
Comment (marcobottaro) [inline apps/nextjs-website/src/app/layout.tsx:98]: By doing this you are going to add these tags to all the pages using the layout and duplicate page-specific meta tags created with the `makeMetadata` helper (see `apps/nextjs-website/src/helpers/metadata.helpers.ts`).
You should use the helper's function in `app/page.tsx` to set the tags in the home page

**Review by marcobottaro — CHANGES_REQUESTED** [review]
Replace the image when it arrives and move meta tags to `app/page.tsx`

**Review by marcobottaro — APPROVED** [review]

File: apps/nextjs-website/src/_contents/appIo/appIo.ts [hunk 2]
```diff
@@ -4,12 +4,14 @@ import { appIOGuideListsPath } from '@/_contents/appIo/guideListsPath';
...
 import { appIoQuickStartGuidePath } from '@/_contents/appIo/quickStartGuidePath';
 import { appIoTutorialListsPath } from '@/_contents/appIo/tutorialListsPath';
+import { baseUrl } from '@/config';
 
 export const appIo: Product = {
   name: "L'App IO",
   description:
     'Raccogli tutti i servizi digitali del tuo ente in un’unica piattaforma e interagisci in modo semplice e sicuro con i cittadini.',
   svgPath: '/icons/appIo.svg',
+  pngPath: `${baseUrl}/icons/appIo.png`,
```
Comment (kin0992) [inline apps/nextjs-website/src/_contents/appIo/appIo.ts:14]: It's not clear to me why a path contains the `baseUrl` and the other doesn't.

File: apps/nextjs-website/src/_contents/send/send.ts [hunk 3]
```diff
@@ -12,6 +12,7 @@ export const send: Product = {
   description:
     'Invia comunicazioni a valore legale con un processo di notificazione gestito interamente dalla piattaforma.',
   svgPath: '/icons/send.svg',
+  pngPath: '/icons/send.png',
```
Comment (kin0992) [inline apps/nextjs-website/src/_contents/send/send.ts:15]: why this `pngPath` does not have the `baseUrl`?

File: apps/nextjs-website/src/app/page.tsx [hunk 4]
```diff
@@ -14,8 +14,10 @@ export async function generateMetadata(): Promise<Metadata> {
   const { homepage } = translations;
 
   return makeMetadata({
-    description: homepage.productsShowcaseTitle,
-    url: '/',
+    title: 'PagoPA DevPortal',
+    description: 'Il portale per gli sviluppatori di PagoPA',
+    url: 'https://developer.pagopa.it',
```
Comment (kin0992) [inline apps/nextjs-website/src/app/page.tsx:19]: should url be equal to `baseUrl`?

File: apps/nextjs-website/src/_contents/appIo/appIo.ts (hunk 2 above)
Comment (marcobottaro) [reply apps/nextjs-website/src/_contents/appIo/appIo.ts:14]: The svgPath is only for internal use, the pngPath is used on the og meta tag which requires an absolute URL. We should rename it as `pngUrl`

Comment (marcobottaro) [reply apps/nextjs-website/src/_contents/appIo/appIo.ts]: It's still used in the cards of the Product Showcase: `apps/nextjs-website/src/components/organisms/ProductsShowcase/ProductsShowcase.tsx`. We cannot remove it!

File: apps/nextjs-website/src/config.ts [hunk 5]
```diff
@@ -24,6 +24,12 @@ export const profileMenuItems: readonly {
   { label: 'agreements.title', href: '/profile/agreements' },
 ];
 
-export const snackbarAutoHideDurationMs = 10_000;
+export const RESET_AFTER_MS = 4_000;
+
```
Comment (kin0992) [inline apps/nextjs-website/src/config.ts:28]:
```suggestion
```

**Review by kin0992 — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]

**Pull request merged**